ecl/baremetal/v2/_proxy.py

```diff
@@ -562,3 +563,10 @@ def get_chassis(self, chassis_id):
         # Use "Chassis" instead of "ChassisDetail".
         # Because "detail" is not included to the request path.
         return self._get(_chassis.Chassis, chassis_id)
+
+    def bmc_reset(self, details=True):
+        bmc_reset = _reset.BmcResetDetail if details else _reset.BmcReset
+        return list(self._list(bmc_reset))
+
+    def get_bmc_reset(self, server_id):
+        return self._get(_reset.BmcReset, server_id)
```

Comment:
```suggestion
        return self._get(_reset.BmcReset, server_id)

```
ファイルの末尾には必ず改行コードを入れてください。

Reply:
ありがとうございます。改行いたしました。
コード整形も使いながら整えていきます。